dask/array/tests/test_creation.py

```diff
@@ -119,9 +120,36 @@ def test_arange():
     nparr = np.arange(0, -1, 0.5)
     assert_eq(darr, nparr)
 
-
-def test_arange_has_dtype():
-    assert da.arange(5, chunks=2).dtype == np.arange(5).dtype
+    # Unexpected or missing kwargs
+    with assert_raises(TypeError):
+        da.arange(10, chunks=-1, unexpected=True)
```

Comment:
It would be nice (but not mandatory) to include the following check:

```python
with pytest.raises(TypeError) as info:
    da.arange(10, chunks=-1, unexpected=True)
assert 'unexpected' in str(info.value)
```